contracts/VToken.sol

```diff
@@ -781,6 +799,11 @@ contract VToken is
         totalBorrows = totalBorrowsNew;
         totalReserves = totalReservesNew;
 
+        if (currentBlockNumber - reduceReservesBlockNumber >= reduceReservesBlockDelta) {
```

Comment:
I wonder what happens if we forget to initialize this variable. Will the reserves be withdrawn every block?

Maybe we'd rather want to treat 0 as "automatic allocation is switched off"?

```suggestion
        uint256 reduceReservesBlockDelta_ = reduceReservesBlockDelta;
        if (reduceReservesBlockDelta_ && currentBlockNumber - reduceReservesBlockNumber >= reduceReservesBlockDelta) {
```

Reply:
@kkirka I believe that initially we won't be missing it anyway because we will add all of the necessary initialization in the VToken upgrades https://github.com/VenusProtocol/vips/pull/67.